pkg/api/types.go

```diff
@@ -150,6 +150,12 @@ type ObjectMeta struct {
 	// from the list. If the deletionTimestamp of the object is non-nil, entries
 	// in this list can only be removed.
 	Finalizers []string `json:"finalizers,omitempty"`
+
+	// A referece to the cluster which the object belongs to.
+	// This is used to distinguish resources with same name and namespace in different clusters.
+	// An empty value means the object belongs to the local cluster (the one the client connects to).
```

Comment:
I think at some point this may stop being true, clusters will present their own IDs.